### PR TITLE
OpsGenie draft

### PR DESCRIPTION
**Pull request by ethangj**
@mlocher Do you know if we have, or if you could quickly write, a very simple script that just shows something like a simple if with a real deployment command (like s3 or heroku)? I think it's optional in the scope of this, and I don't see one sitting around, was just curious if we had anything prepared already to bump this up a bit in depth.

### CODE REVIEW

**Comment by mlocher** [comment]
Assuming you want to create a script to create an OpsGenie alert, they have a basic `curl` command available at https://www.opsgenie.com/docs/web-api/alert-api#createAlertRequest

We could either add that (in a modified way) to the documentation, or if we want to, add it to codeship/scripts as well.

**Comment by ethangj** [comment]
@mlocher what about the conditional, showing only run the curl if something else fails? do we have an example on hand for that? probably not strictly necessary.

**Comment by mlocher** [comment]
@ethangj closest we get is https://github.com/codeship/scripts/blob/master/utilities/ensure_called.sh

**Comment by ethangj** [comment]
@mlocher kk, will just go with the opsgenie example then, can you go ahead and review PR

File: _general/integrations/opsgenie.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  - reporting
+  - integrations
+---
+
+* include a table of contents
+{:toc}
+
+## About OpsGenie
+
+[OpsGenie](https://www.opsgenie.com) lets send alerts and notifications about critical events to your on-call or development team.
```
Comment (mlocher) [inline _general/integrations/opsgenie.md]: lets _you_ send ...

File: _general/integrations/opsgenie.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+[Their documentation](https://www.opsgenie.com/docs) does a great job of providing more information, in addition to the setup instructions below.
+
+## Codeship Pro
+
+### Setting Your API Key
+
+You will need to add your OpsGenie API key to your [encrypted environment variables]({{ site.baseurl }}{% link _pro/builds-and-configuration/environment-variables.md %}) that you define in your [codeship-services.yml file]({{ site.baseurl }}{% link _pro/builds-and-configuration/services.md %}).
```
Comment (mlocher) [inline _general/integrations/opsgenie.md]: that sentence sounds like it's missing something? The encrypted environment variables are not defined in the services file?

File: _general/integrations/opsgenie.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+### Sending Alerts
+
+Next, you will need to add [API calls to OpsGenie](https://www.opsgenie.com/docs/rest-api/alert-api), in your [codeship-steps.yml file]({{ site.baseurl }}{% link _pro/builds-and-configuration/steps.md %}), most likely to alert of a successful deployment or combined with your deployment commands in a script to alert of a failed deployment.
```
Comment (mlocher) [inline _general/integrations/opsgenie.md]: I would simplify that sentence and split it in two, one for each of the possible ways to integrate this.

**Pull request merged**